### PR TITLE
Add icons for cart and checkout screen

### PR DESCRIPTION
**Pull request by Alex7Li**
This commit is too big, it does a couple of things. It implements tooltips to delete items from the cart and show a codemodal with analytics code. It also fires analytics code. Prior to this commit, you can't remove items from the cart, and can only set quantity to 0, justifying the delete button. This commit is the first to show any codemodals, and I spent some time working on the css to make the tooltip look better.

Analytics code won't work until #25 is merged

Screenshot 2020-06-18 at 12 10 42 PM
Screenshot 2020-06-18 at 12 29 27 PM
Screenshot 2020-06-18 at 12 14 17 PM
Screenshot 2020-06-18 at 12 13 39 PM

### CODE REVIEW

**Comment by Alex7Li** [comment]
I had a really tiny PR dependent on this one, (adding correct code modal text and firing events using helper functions), and this PR had some big merge conflicts with a recent push, so when I was fixing the merge conflicts I just put all the code together here instead of waiting for this to get accepted, then sending out another small PR.

File: demo/src/components/Cart/Cart.js [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+import {getRemoveFromCartCodeSnippet, getViewCartCodeSnippet} from '../../lib/gtagSnippets';
+import {sendRemoveFromCartEvent} from '../../lib/gtagEvents';
 
 /**
  * Creates a component describing a shopping Cart.
  * @param {ItemStore} items The global {@link ItemStore} site object.
  * @param {function(string, number)} setQuantity A function to modify
  *      the quantity of an item in the global state.
+ * @param {function(string)} removeOneFromCart A function to reduce the
+ * quantity of an element in the cart by 1, deleting it if the quantity is
```
Comment (durbanie) [inline demo/src/components/Cart/Cart.js:20]: Please fix the indentation.

File: demo/src/components/Cart/Cart.js [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+ * empty.
  * @return {!JSX} The component.
  */
-const CartBase = function({items, setQuantity}) {
+const CartBase = function({items, setQuantity, removeOneFromCart}) {
```
Comment (durbanie) [inline demo/src/components/Cart/Cart.js:24]: nit: I'd prefer arrow functions when possible. Is there a reason not to use one here?

Comment (Alex7Li) [reply demo/src/components/Cart/Cart.js]: Not that I know of, I changed it.

File: demo/src/components/Cart/Cart.js (hunk 1 above)
Comment (Alex7Li) [reply demo/src/components/Cart/Cart.js]: Fixed

**Pull request merged**